packages/foreman/rubygem-concurrent-ruby/rubygem-concurrent-ruby.spec

```diff
@@ -63,10 +63,13 @@ cp -a .%{gem_dir}/* \
 
 %files
 %dir %{gem_instdir}
-%doc %{gem_instdir}/LICENSE.txt
+%doc %{gem_instdir}/LICENSE.md
+%doc %{gem_instdir}/Gemfile
+%doc %{gem_instdir}/Rakefile
```

Comment:
I wonder if this can't be excluded instead. Same for the Gemfile.

Reply:
updated